frontend/src/scenes/notebooks/Notebook/NotebookIconHeading.tsx

```diff
@@ -0,0 +1,9 @@
+const NotebookIconHeading = ({ level }: { level: number }): JSX.Element => {
```

Comment:
I think this is actually the way to go. Not sure we _need_ icons for stuff that is essentially text